src/orb.yml.hbs

```diff
@@ -1145,6 +1147,7 @@ commands:
                 set -- "$@" --network-configuration awsvpcConfiguration="{subnets=[<<parameters.subnet-ids>>],securityGroups=[<<parameters.security-group-ids>>],assignPublicIp=<<parameters.assign-public-ip>>}"
             fi
 
+            echo "Before run-task \$@ is: $@"
```

Comment:
I would like the debug output here to be toggleable as well (see my previous comment)